src/klein/_session.py

```diff
@@ -123,7 +123,7 @@ def procureSession(
             ]
             # Does it seem like this check is expensive? It sure is! Don't want
             # to do it? Turn on your dang HTTPS!
-            yield self._store.sentInsecurely(allPossibleSentTokens)
+            self._store.sentInsecurely(allPossibleSentTokens)
```

Comment:
`sentInsecurely()` returns `None` and is not async, so mypy complains about this unnecessary `yield`.